### PR TITLE
messageLinkEmbeds: fix AutoModEmbed on compact mode

### PR DESCRIPTION
**Pull request by lumap**
Tried to find a solution with the Embed type but I didn't find anything

### CODE REVIEW

File: src/plugins/messageLinkEmbeds.tsx [hunk 1]
```diff
@@ -336,7 +337,7 @@ function AutomodEmbedAccessory(props: MessageEmbedProps): JSX.Element | null {
                 <span>{isDM ? " - Direct Message" : " - " + GuildStore.getGuild(channel.guild_id)?.name}</span>
             </Text>
         }
-        compact={false}
+        compact={getSettingStoreLazy<boolean>("textAndImages", "messageDisplayCompact")?.getSetting() || false}
```
Comment (Vendicated) [inline src/plugins/messageLinkEmbeds.tsx:340]: this should be stored inside a variable on the top level

Comment (lumap) [reply src/plugins/messageLinkEmbeds.tsx]: did i do it right

Comment (Vendicated) [reply src/plugins/messageLinkEmbeds.tsx]: no, that is not top level. you should store the settings store globally

File: src/plugins/messageLinkEmbeds.tsx [hunk 2]
```diff
@@ -336,7 +338,7 @@ function AutomodEmbedAccessory(props: MessageEmbedProps): JSX.Element | null {
                 <span>{isDM ? " - Direct Message" : " - " + GuildStore.getGuild(channel.guild_id)?.name}</span>
             </Text>
         }
-        compact={false}
+        compact={compactModeEnabled?.getSetting() || false}
```
Comment (Vendicated) [inline src/plugins/messageLinkEmbeds.tsx:341]:
```suggestion
        compact={compactModeEnabled.getSetting()}
```

**Pull request closed without merging**